pkg/cosign/remote.go

```diff
@@ -80,7 +83,14 @@ func findDuplicate(ctx context.Context, sigImage v1.Image, payload []byte, dupeD
 		return nil, err
 	}
 
+LayerLoop:
 	for _, layer := range manifest.Layers {
+		// if there are any new annotations, then this isn't a duplicate
```

Comment:
@dekkagaijin does this look ok to you? basically I needed cosign to know that if we are trying to add a new bundle annotation then don't mark this new signature as a duplicate

Reply:
hrm. we'd probably also want to verify that the values are the same (for annotations other than `signature`)

Reply:
for sure